### PR TITLE
Changes to show SPA/PWA tab for no data

### PR DESCRIPTION
**Pull request by AltamashShaikh**
### Description:

Changes to show SPA/PWA tab for no data screen.
Fixes: #PG-2845

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: vue/src/TagmanagerTrackingCode/TrackingSPAPage.vue [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      :showStep2="false"
+      :showStep3="false"
+      @fetchInstallInstructions="fetchInstallInstructionsSPA"
+      ref="trackingCodeCommon"
+    />
+    <li v-html="$sanitize(setupStep2)"></li>
+    <li v-html="$sanitize(setupStep3)"></li>
+    <li v-text="translate('TagManager_SPAFollowStep4')"></li>
+    <li v-text="translate('TagManager_SPAFollowStep5')"></li>
+    <li v-html="$sanitize(fetchFollowStepCommon(5))"></li>
```
Comment (snake14) [inline vue/src/TagmanagerTrackingCode/TrackingSPAPage.vue:22]: I think this is supposed to be `6`? When I test locally, it's showing the same text for steps 5 and 6.

**Review by snake14 — CHANGES_REQUESTED** [review]
Everything looks good. When I checkout the corresponding core branch, everything displays as expected.

I only noticed one issue. Please see my comment.

File: vue/src/TagmanagerTrackingCode/TrackingSPAPage.vue (hunk 1 above)
Comment (AltamashShaikh) [reply vue/src/TagmanagerTrackingCode/TrackingSPAPage.vue:22]: @snake14 Updated points now point 4 and 6 is same as we have same header for both the buttons

**Review by snake14 — APPROVED** [review]
It's a little odd that there's no step 6 translation, but it works as expected :+1:

**Comment by snake14** [comment]
Looks like a few expected results need to be updated for the PHP and UI builds.

**Pull request merged**